aspnetcore/fundamentals/app-state/sample/src/WebAppSession/Controllers/HomeController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace WebAppSession.Controllers
+{
+    #region snippet1
+    public class HomeController : Controller
+    {
+        const string SessionKeyName = "_SessionKeyName";
```

Comment:
Removed, but I did find it reassuring when I examined the value in the debugger :)